docusaurus/docs/how-to-guides/panel-plugins/plugin-internationalization.md

```diff
@@ -0,0 +1,306 @@
+---
+id: plugin-internationalization
+title: How to add translations to your plugin
+description: Translate your plugins
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - frontend
+  - tooling
+  - configuration
+  - translation
+  - localization
+  - internationalization
+  - webpack
```

Comment:
nit: dunno what weight keywords bring these days but this doc doesn't feel too webpack specific to me so would consider dropping from this list.